src/message.ts

```diff
@@ -613,9 +721,44 @@ export class Message implements Sayable {
   // }
 
   /**
-   * @todo document me
+   * Reply a text message to the sender.
+   *
+   * @param {string} text
+   * @param {(Contact | Contact[])} [replyTo]
+   * @returns {Promise<any>}
+   * @example
+   * ```ts
+   * const bot = Wechaty.instance()
+   * bot
+   * .on('message', async m => {
+   *   if (/^(ding|ping|bing|code)$/i.test(m.content())) {
```

Comment:
This `if` block is not related the example. 

Should consider deleting those codes here.